docker/nginx/docker-compose.yml

```diff
@@ -113,3 +113,9 @@ services:
     deploy:
       restart_policy:
         condition: none
+
+  cl-doctor:
+    image: freelawproject/doctor:latest
+    container_name: "cl-doctor"
+    networks:
+      - cl_net_overlay
```

Comment:
You don't need to expose ports here, like the others do? 

What about logging?

Reply:
That's the last thing. When you're happy with these two comments, it's merging time!

Reply:
Oh, except `black` is unhappy again. :(